_posts/2020-04-29-newpipe-0.19.3-release.md

```diff
@@ -0,0 +1,61 @@
+
+## YouTube music
+
+Since 2015, the YouTube music app exists; since 2018, a web app is there. As of today, you can search for content of this music platform in NewPipe. This looks a little something like this:
+
+<img class="no-flow img-responsive" src="/img/screenshots/shot_15.png"/>
+
+[@wb9688](https://github.com/wb9688) is responsible for integrating this into the Extractor (the core module for getting the content from the platforms) and in the Android App, big thanks for allowing us to indulge in our music habits as much as we like without having reaction videos jumping in our search results. 
+
+
+## Bug fixes
+
+Imagine you would see a button where you would not expect one to be. Sounds scary, hmm? [@mauriciocolli](https://github.com/mauriciocolli) took care of it. It won't hurt anyone anymore. He also improved the error experience. Even though we obviously don't want you to see any, now they are the right ones at least. [@wb9688](https://github.com/wb9688) (if you think you will read this name a lot of times in this post, you are not mistaken) fixed an issue where age restricted videos were shown even though the related option was turned off.
+
+[@B0pol](https://github.com/B0pol) fixed the generation of crash reports in the background by using a different JSON library which doesn't provide invalid JSON anymore in some cases. A temporary fix for bookmarked playlists which names changed to null was provided by [@mitosagi](https://github.com/mitosagi). Now you can null away all you want. [@Stypox](https://github.com/Stypox) tried one more time to fix the handling of ReCaptchas for some more types of them and is probably questioning his sanity at this point. Thank you though!
+
+## Improvements
+
+ [@wb9688](https://github.com/wb9688) improved the background check for new versions, it uses the same downloader as other functions of the app now, which will make adding more features to it in the future easier. [@mauriciocolli](https://github.com/mauriciocolli) added code in his third contribution to avoid unnecessary changes to upload dates of saved streams, so they do not jump around in your bookmarked playlists anymore. Thanks to [@XiangRongLin](https://github.com/XiangRongLin), the playback parameters you change in the options are now applied in every player, which leads to a more unified experience. The third contribution of [@B0pol](https://github.com/B0pol) greatly improves the user experience when NewPipe faces unsupported content. Instead of crashing, you will see a big kaomoji now. Definitly an improvement.
```

Comment:
And JSON parser.
Definitely

Reply:
in the quoted content, nothing mentions JSON. Not sure where you want to have this?

Reply:
>it uses the same downloader

Reply:
Also please mention somewhere that we're now using our fork of nanojson instead of upstream nanojson. Difference is that it supports e.g. `{this: 'kind', of: 'json'}` instead of `{"just":"this"}`, which is needed for e.g. Bandcamp support that will be coming in the future.